### PR TITLE
TMA-785: Support excludeFactRule parameter

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/761/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/625/ : SUCCESS in 1m 53s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/886/ : SUCCESS in 2m 03s

File: lib/gooddata/lcm/actions/synchronize_ldm.rb [hunk 1]
```diff
@@ -26,15 +26,21 @@ class SynchronizeLdm < BaseAction
...
       class << self
         def call(params)
           # set default value for include_computed_attributes
-          # (we won't have to do this after TMA-690)
+          # and exclude_fact_rule (we won't have to do this after TMA-690)
           include_ca = params.include_computed_attributes
           include_ca = true if include_ca.nil?
           include_ca = include_ca.to_b
+          exclude_fact_rule = params.exclude_fact_rule
+          exclude_fact_rule = false if exclude_fact_rule.nil?
```
Comment (kubamahnert) [inline lib/gooddata/lcm/actions/synchronize_ldm.rb]: the default functionality should be already implemented in TMA-690, isn't this superfluous?

File: lib/gooddata/lcm/actions/synchronize_ldm.rb [hunk 2]
```diff
@@ -26,15 +26,15 @@ class SynchronizeLdm < BaseAction
...
       end
 
       class << self
         def call(params)
-          # set default value for include_computed_attributes
-          # (we won't have to do this after TMA-690)
-          include_ca = params.include_computed_attributes
-          include_ca = true if include_ca.nil?
-          include_ca = include_ca.to_b
+          include_ca = params.include_computed_attributes.to_b
```
Comment (yenkins) [inline lib/gooddata/lcm/actions/synchronize_ldm.rb]: Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/763/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/626/ : SUCCESS in 55s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/887/ : FAILURE in 43s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/767/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/629/ : FAILURE in 8s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/890/ : SUCCESS in 34s

**Comment by kubamahnert** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/768/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/630/ : SUCCESS in 2m 31s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/891/ : SUCCESS in 2m 20s

**Comment by kubamahnert** [comment]
Good job!

**Comment by kubamahnert** [comment]
ok to merge

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/289/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/632/ : SUCCESS in 34s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/893/ : SUCCESS in 35s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1220/ : SUCCESS in 52m 32s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/978/ : UNSTABLE in 4m 02s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/298/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/641/ : SUCCESS in 1m 56s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/902/ : SUCCESS in 1m 46s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1229/ : FAILURE in 2m 09s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/987/ : SUCCESS in 35m 46s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/300/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/643/ : SUCCESS in 43s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/904/ : SUCCESS in 36s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1231/ : FAILURE in 49m 34s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/989/ : UNSTABLE in 12m 16s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/305/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/656/ : SUCCESS in 33s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/917/ : SUCCESS in 35s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1238/ : FAILURE in 59m 22s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/996/ : UNSTABLE in 7m 21s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/306/ : SUCCESS in 22s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/657/ : SUCCESS in 2m 19s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/918/ : SUCCESS in 2m 19s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1241/ : SUCCESS in 1h 30m 06s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/999/ : ABORTED in 1h 00m 02s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/307/ : SUCCESS in 26s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/658/ : SUCCESS in 36s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/919/ : SUCCESS in 35s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1242/ : FAILURE in 50m 42s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1000/ : UNSTABLE in 39m 42s

**Pull request merged**